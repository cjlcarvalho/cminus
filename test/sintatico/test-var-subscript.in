void main(void)
{
    x[0];
    x[1];
    x[99];
    x[y];
    x[1 + y * (z - w)];
}
