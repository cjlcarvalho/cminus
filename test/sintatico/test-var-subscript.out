[program 
  [decfunc 
    [void]
    [main]
    [paramlist]
    [block [var x [0]] [var x [1]] [var x [99]] [var x [var y]] [var x 
        [+ [1] 
          [* [var y] 
            [- [var z] [var w]]]]] 
    ]
  ]
]
